.env.development

```diff
@@ -0,0 +1 @@
+VITE_ZONE_CREATION_FLOW=enabled
```

Comment:
I’m avoiding "true" and "false" for .env entries on purpose because usually, .env file entry values are parsed as string and you’d need to check against `'true'` and `'false'` anyway. To not suggest that this will contain a true boolean, I’m using "enabled".

In any case, it works the same so I’m leaving this as is.

Reply:
> you’d need to check against 'true' and 'false' anyway.

We already use `"true"|"false"` to determine whether something is on/off, enabled/disabled, yes/no. Its just easier to stick to the same combo for everything.

Once we use less strings in `env` use pretty much just `booleans` we can make `env` return only a boolean-ish instead of only strings. Now we are adding a `i18n` layer that is closer to happening.

🤔 In fact I'd actually prefer `"1"|""` I think. The only env var we have using `"true"|"false"` currently is the MSW enabling one right?

Lemme know, I think I'm preferring to change this to `"1"|""` but I'm not totally sure if the MSW enabling var is the only one that currently uses `"true"|"false"`